wolfcrypt/src/random.c

```diff
@@ -3515,6 +3535,26 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
      * extern int myRngFunc(byte* output, word32 sz);
      */
 
+#elif defined(__MICROBLAZE__)
+    #warning weak source of entropy
+    #define LPD_SCNTR_BASE_ADDRESS 0xFF250000
+
+    int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
+    {
+        word32* cnt;
+        word32 i;
+
+        /* using current time with srand */
+        cnt = (word32*)LPD_SCNTR_BASE_ADDRESS;
+        srand(*cnt | *(cnt+1));
```

Comment:
This isn't a great way to generate cryptographic seeds.
Is there nothing else?

Reply:
The device did not have a TRNG source. I can remove this from the pull request if would like to avoid the weak entropy source. Would go to using WOLFSSL_GENSEED_FORTEST instead.

Reply:
We do this for other cases so I'm happy to use this.